core/templates/dev/head/css/oppia.css

```diff
@@ -732,9 +733,15 @@ textarea {
   list-style-position: inside;
   list-style-type: none;
 }
-.oppia-footer-social a:hover {
+.oppia-footer-social a:hover,
+.oppia-footer-social a:focus {
   text-decoration: none;
 }
+.oppia-footer-social a:focus > img,
+.protractor-test-i18n-language-selector:focus {
```

Comment:
Don't use .protractor-test-* classes in CSS -- those are only meant to be for e2e tests. Feel free to invent a new class name if you need to.